cirq/contrib/acquaintance/gates.py

```diff
@@ -309,7 +310,8 @@ def _decompose_(self, qubits: Sequence['cirq.Qid']) -> 'cirq.OP_TREE':
         if final_gate:
             yield final_gate(*qubits)
 
-    def _circuit_diagram_info_(self, args: 'cirq.CircuitDiagramInfoArgs'):
+    def _circuit_diagram_info_(self, args: 'cirq.CircuitDiagramInfoArgs'
+                              ) -> 'cirq.CircuitDiagramInfo':
```

Comment:
How come one of the _circuit_diagram_info_ returns Iterable[str] and the other returns cirq.CircuitDiagramInfo?

I forget.  Does this protocol allow both?

Reply:
Yes, mypy supports return type covariance and the protocol supports `str`, `Iterable[str]` and `CircuitDiagramInfo`, see [here](https://github.com/quantumlib/Cirq/blob/master/cirq/protocols/circuit_diagram_info_protocol.py#L182). In this case I chose the most concrete type because we don't derive any subclasses from these.